build.gradle

```diff
@@ -388,8 +394,10 @@ subprojects { subproject ->
     implementation group: 'javax.inject', name: 'javax.inject', version: '1'
     implementation group: 'javax.jws', name: 'javax.jws-api', version: '1.1'
 
-    implementation group: 'org.apache.cxf', name: 'cxf-rt-frontend-jaxws', version: cxfVersion
-    implementation group: 'org.apache.cxf', name: 'cxf-rt-transports-http', version: cxfVersion
+    implementation group: 'org.apache.cxf', name: 'cxf-spring-boot-starter-jaxws', version: cxfVersion
+    implementation group: 'org.apache.cxf', name: 'cxf-rt-transports-http-jetty', version: cxfVersion
+//    implementation group: 'org.apache.cxf', name: 'cxf-rt-transports-http', version: cxfVersion
```

Comment:
Remove this commented out code if it isn't needed.

Reply:
done